src/test/java/io/vertx/test/core/HttpTest.java

```diff
@@ -80,13 +81,22 @@
   public TemporaryFolder testFolder = new TemporaryFolder();
 
   protected File testDir;
+  protected SocketAddress testAddress;
+  private File tmp;
 
   @Override
   public void setUp() throws Exception {
     super.setUp();
+    if (USE_DOMAIN_SOCKETS) {
```

Comment:
actually we don't need this because it cannot work since we only have HTTP server with domain sockets and not HTTP client, that's why only a simple test is necessary